main.go

```
@@ -217,11 +261,19 @@ func outputCSV(accounts []AccountInfo) {
 	defer writer.Flush()
 
 	// Write header
-	writer.Write([]string{"alias_name", "account_id"})
+	writer.Write([]string{"id", "arn", "email", "name", "status", "joined_method", "joined_timestamp"})
```

golangci-lint (1.64.8)

Error return value of `writer.Write` is not checked

(errcheck)